### PR TITLE
62075 - Added a widget to display an info alert on the how to apply p…

### PR DESCRIPTION
**Pull request by fred-khoury-oddball**
## Summary

- _(Summarize the changes that have been made to the platform)_
  - Added a new widget that is going to be used on the pension how to apply page.
- _(If bug, how to reproduce)_
  - N/A
- _(What is the solution, why is this the solution)_
  - The solution is a react widget that can be used on the burial how to apply page. This is the solution because it is a widget meeting the requested requirements that can be used on the pension how to apply page.
- _(Which team do you work for, does your team own the maintenance of this component?)_
  - Benefits and Non Disability Claims
- _(If using a flipper, what is the end date of the flipper being required/success criteria being targeted)_
  - N/A

## Related issue(s)

- _Link to ticket created in va.gov-team repo_
https://github.com/department-of-veterans-affairs/va.gov-team/issues/64136
- _Link to requirements_
https://github.com/department-of-veterans-affairs/va.gov-team/issues/62448

## Testing done

- _Describe what the old behavior was prior to the change_
 - There was no widget prior to the change
- _Describe the steps required to verify your changes are working as expected_
  - Navigate to /pension-how-do-i-apply-widget endpoint
  - View the widget should be loaded on the page 
  - Content should reflect the unauthenticated state
  - Login
  - Navigate to /pension-how-do-i-apply-widget endpoint
  - Widget should reflect authenticated content
- _Describe the tests completed and the results_
   - Unit tests were written and passing.
     - We tested the content of the widget when authenticated and unauthenticated
   - E2E 
    -  We wrote cypress tests
      - We tested both authenticated and unauthenticated state
      - We tested to ensure the widget is accessible 
    - Manual
      - We manually tested this component by navigating to /pension-how-do-i-apply-widget endpoint and ensuring it was working as expected.
  - Accessibility
    - We tested using a screen reader and keyboard to navigate the widget.

#### Unauth
image
#### Auth
image

#### Unauth
image
image

#### Auth
image

#### Unauth
image
image
image

#### Auth
image
using
image
image

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|         | Before | After |
| ------- | ------ | ----- |
| Mobile  |        |       |
| Desktop |        |       |

## What areas of the site does it impact?
This is a widget and should be isloted and not impact any other parts of the site unless explicitly implemented.

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [x] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added
- [x] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [x] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [x] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_ The new endpoints should not be available in prod. I don't think it should be built in staging or prod but was wondering if I could get confirmation that.

### CODE REVIEW

**Review by tblackwe — APPROVED** [review]
LGTM

**Review by rsmithadhoc — APPROVED** [review]

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -66,6 +66,7 @@ src/applications/burials @department-of-veterans-affairs/vsa-debt-frontend @depa
 src/applications/financial-status-report @department-of-veterans-affairs/vsa-debt-frontend
 src/applications/medical-copays @department-of-veterans-affairs/vsa-debt-frontend
 src/applications/combined-debt-portal @department-of-veterans-affairs/vsa-debt-frontend
+src/applications/static-pages/pension-how-do-i-apply-widget @department-of-veterans-affairs/benefits-ce-nondisability
```
[the remark this answers is not in the document]
Comment (tblackwe) [reply .github/CODEOWNERS:73]: https://github.com/orgs/department-of-veterans-affairs/teams/benefits-ce-nondisability the team is there. curious what needs to be done to grant access

[the remark this answers is not in the document]
Comment (fred-khoury-oddball) [reply .github/CODEOWNERS:73]: Yea I was wrong our team does exist. I am not sure how to get the squiggles to disappear but I am investigating this.

[the remark this answers is not in the document]
Comment (fred-khoury-oddball) [reply .github/CODEOWNERS]: Our team does not have access to this repository. We are removing this line until we get access. 
@powellkerry @tblackwe Thank you.

**Pull request merged**